Configuration/PyReleaseValidation/README.md

```diff
@@ -107,6 +107,8 @@ The offsets currently in use are:
 * 0.203: HGCAL TICLv5
 * 0.204: HGCAL superclustering : using Mustache in TICLv5
 * 0.205: HGCAL superclustering : using old PFCluster-based Mustache algorithm with TICLv5
+* 0.209: TICL barrel : run TICL in HGCAL and barrel calorimeters
+* 0.2091: TICL barrel with CaloParticles from PU enabled
```

Comment:
can you please document *also* the `0.758` offset?

Reply:
another small request, can you add `29634.758` to the `ph2_hlt` short matrix, see https://github.com/cms-sw/cmssw/pull/48262 for an example.

Reply:
@brusale can you address this comment too?